index.html

```diff
@@ -12,88 +12,196 @@
     />
   </head>
   <body>
-    <div class="container">
-      <section class="Content">
-        <h1 class="about-me">About Myself!</h1>
+    <!-- First Figma-->
+    <section class="card">
+      <img src="Asset/Tonic.png" alt="Tonic Company" class="tonic-image" />
+      <div class="TonicHeader">
+        <p>Tonic</p>
+      </div>
+      <div class="HeaderTitle">
+        <p class="tonic-title">CANOPY</p>
+        <img src="Asset/Counter.png" class="counter" />
+        <p class="TonicEnd">Back End Dev</p>
+        <img src="Asset/Counter.png" class="counter" />
+        <p class="TonicEnd">2015</p>
+      </div>
+      <div class="Content">
+        <p>
+          a daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+      </div>
+      <div class="ButtonGroup">
+        <button>html</button> <button>css</button> <button>javaScript</button>
+      </div>
+      <div class="See-more">
+        <button>See Project</button>
+      </div>
+    </section>
+    <!--Second Figma-->
+    <section class="card">
+      <img src="Asset/MultiPost.png" alt="Tonic Company" class="tonic-image" />
+      <div class="TonicHeader">
+        <p>Multi-Post Stories</p>
+      </div>
```

Comment:
- [ ] The project title (e.g., "Tonic") should use a header tag `hx` instead of `p` tag for semantic purposes.
- [ ] Also unless you need to include the `div `for some structural/organizational reason you're likely better off just styling the `hx`class. The purpose of `divs` is to allow you to group elements together. Wrapping individual elements in `divs` for the purpose of styling is largely unnecessary.